### PR TITLE
Weapon effects

### PR DESCRIPTION
**Pull request by JFranklin3000**
##### In this PR
- Created Weapon Effects template
- Created Weapon Effects Item sheet

##### Testing
- Create an Effect Item 
- Validated fields working,

### CODE REVIEW

File: lang/en.json [hunk 1]
```diff
@@ -278,6 +285,7 @@
     "ShiftAutoSuccess": "Auto Success",
     "SkillSelector": "Select Skills",
     "ShiftCriticalSuccess": "Critical Success",
+    "ShiftDown": "Shift Down",
```
Comment (phildominguez) [inline lang/en.json:288]: How about add ShiftUp and remove RollDialogShiftDown/RollDialogShiftUp?

File: lang/en.json [hunk 2]
```diff
@@ -19,6 +19,7 @@
       "bond": "Bond",
       "classFeature": "Class Feature",
       "contact": "Contact",
+      "effect": "Weapon Effect",
```
Comment (phildominguez) [inline lang/en.json:22]: Any objections to making it `weaponEffect`? Right now it's out of order in the item creation menu.

**Review by phildominguez — CHANGES_REQUESTED** [review]

File: lang/en.json (hunk 2 above)
Comment (JFranklin3000) [reply lang/en.json]: nope changed.

File: lang/en.json (hunk 1 above)
Comment (JFranklin3000) [reply lang/en.json:287]: done.

**Review by phildominguez — APPROVED** [review]

**Pull request merged**